Card/src/main/java/com/paypal/android/card/network/CardAPIClient.kt

```diff
@@ -0,0 +1,42 @@
+package com.paypal.android.card.network
+
+import com.paypal.android.card.Card
+import com.paypal.android.card.CardError
+import com.paypal.android.card.CardResponse
+import com.paypal.android.card.CardResult
+import com.paypal.android.core.APIClient
+import org.json.JSONObject
+import java.net.HttpURLConnection.HTTP_OK
+
+class CardAPIClient(private val api: APIClient) {
+
+    suspend fun confirmPaymentSource(orderId: String, card: Card): CardResult {
+        val path = "v2/checkout/orders/$orderId/confirm-payment-source"
+        val body = """
+            {
+                "payment_source": {
+                    "card": {
+                        "number": "${card.number}",
+                        "expiry": "${card.expiry}"
```

Comment:
@jcnoriega brought this up in our channel, but I think having a way to serialize request and response objects will be easier to maintain than raw strings of JSON.

Reply:
i think we need to define the idea of just copying Braintree's SDK networking layer (that has Json Parsing) and put it in our SDK.
But in this case in particular, with @sshropshire we thought that if the queries are small, we could consider just putting the raw json like this. I know its not pretty code wise, BUT it almost looks like documentation, anybody who reads this, knows exactly the body of this API. Of course this is not super scalable for huge requests, but it might we worth perhaps sacrificing some code styling for better "readability". Does that make sense?

Reply:
Yeah this made the most sense to me because of the API knowledge gaps. We did this initially to get it to work, but I never thought about using heredoc template strings; they're super-expressive of intent here. There's a strong correlation with Postman as well.

Reply:
Adding additional endpoints to the sdk in the short term would be super simple, and we can provide more abstraction once the requirements become clear.

Reply:
That sounds good. I think for more complex request bodies we'll need to find a more scalable solution. I'm good with merging this in as is though since this call is pretty basic. 

I was thinking for optional params sent in the request body, things might get a little messier than just using string interpolation.

Reply:
Yeah I agree Jose voiced the same concern here. I also thought about the case where we may have to iterate over a collection of items to produce a `POST` body. The main thing here is postponing the additional abstraction layer until we need it. There are a few knowledge gaps on the API side that'll become clear as we add more features.